pkg/ruler/tokens

```diff
@@ -0,0 +1 @@
+��z�
```

Comment:
Is this file expected to be committed? (same for `pkg/ring/tokens`)

Reply:
I don't this is needed anymore?

Reply:
Looks like this is being created in some test. Will find it out and use a temporary dir instead.